src/widgets/forms.js

```diff
@@ -200,16 +235,193 @@ forms.field[UI.ns.ui('Multiple').uri] = function (
+        console.error(msg)
+      }
+      refresh() // 20191213
+    }
+  }
+
+  /** Make a dom representation for an item
+   * @param {Event} anyEvent if used as an event handler
+   * @param {Node} object The RDF object to be represented by this item.
+   */
+  function renderItem (object) {
+    async function deleteThisItem () {
+      if (ordered) {
+        console.log('pre delete: ' + debugString(list.elements))
+        for (let i = 0; i < list.elements.length; i++) {
+          if (list.elements[i].sameTerm(object)) {
+            list.elements.splice(i, 1)
+            await saveListThenRefresh()
+            return
+          }
+        }
+      } else {
+        // unordered
+        if (kb.holds(subject, property, object)) {
+          var del = [$rdf.st(subject, property, object, store)]
+          kb.updater.update(del, [], function (uri, ok, message) {
+            if (ok) {
+              body.removeChild(subField)
+            } else {
+              body.appendChild(
+                error.errorMessageBlock(
+                  dom,
+                  'Multiple: delete failed: ' + message
+                )
+              )
+            }
+          })
+        }
+      }
+    }
+
+    /** Move the object up or down in the ordered list
+     * @param {Event} anyEvent if used as an event handler
+     * @param {Boolean} upwards Move this up (true) or down (false).
+     */
+    async function moveThisItem (event, upwards) {
+      // @@ possibly, allow shift+click to do move to top or bottom?
+      console.log('pre move: ' + debugString(list.elements))
+      for (var i = 0; i < list.elements.length; i++) {
+        // Find object in array
+        if (list.elements[i].sameTerm(object)) {
+          break
+        }
+      }
+      if (i === list.elements.length) {
+        alert('list move: not found element for ' + object)
+      }
+      if (upwards) {
+        if (i === 0) {
+          alert('@@ boop - already at top   -temp message') // @@ make boop sound
+          return
+        }
+        list.elements.splice(i - 1, 2, list.elements[i], list.elements[i - 1])
+      } else {
+        // downwards
+        if (i === list.elements.length - 1) {
+          alert('@@ boop - already at bottom   -temp message') // @@ make boop sound
+          return
+        }
+        list.elements.splice(i, 2, list.elements[i + 1], list.elements[i])
+      }
+      await saveListThenRefresh()
+    }
+    /* A subField has been filled in
+    *
+    * One possibility is to not actually make the link to the thing until
+    * this callback happens to avoid widow links
+     */
+    function itemDone (uri, ok, message) {
+      console.log(`Item ${uri} done callback for item ${object.uri.slice(-7)}`)
+      if (!ok) { // when does this happen? errors typically deal with upstream
+        console.error('  Item done callback: Error: ' + message)
+      } else {
+        linkDone(uri, ok, message)
+      }
+      /*
+      var ins, del
```

Comment:
In general we want to remove these blocks of code when we don't need them anymore. We have it in Git history anyway if we want to retrieve them later on. Having them here makes the code harder to scan.